api/operator/v1/vmanomaly_types.go

```diff
@@ -103,15 +103,18 @@ type VMAnomalySpec struct {
 	vmv1beta1.CommonApplicationDeploymentParams `json:",inline,omitempty"`
 }
 
-// VMAnomalyWritersSpec defines writer configuration for VMAnomaly
-type VMAnomalyWritersSpec struct {
+// VMAnomalyWriterSpec defines writer configuration for VMAnomaly
+type VMAnomalyWriterSpec struct {
 	// DatasourceURL defines remote write url for write requests
 	// provided endpoint must serve /api/v1/import path
 	// vmanomaly joins datasourceURL + "/api/v1/import"
 	DatasourceURL string `json:"datasourceURL" yaml:"datasource_url,omitempty"`
 	// Metrics to save the output (in metric names or labels)
 	// +optional
 	MetricFormat VMAnomalyVMWriterMetricFormatSpec `json:"metricFormat,omitempty" yaml:"metric_format,omitempty"`
+	// Optional, to override the default import path
+	// +optional
+	ImportJsonPath string `json:"importJsonPath,omitempty" yaml:"import_json_path,omitempty"`
```

Comment:
Looks like it's deprecated:
```
            logger.warning(
                'The use of custom import path is deprecated. '
                'Please use tenant_id instead for automatic path generation.'
            )
```
cc @fred-navruzov

Reply:
haven't see any note regarding this in docs